OpenRA.Game/Traits/Player/FixedColorPalette.cs

```diff
@@ -17,11 +17,13 @@ namespace OpenRA.Traits
 	[Desc("Add this to the World actor definition.")]
 	public class FixedColorPaletteInfo : ITraitInfo
 	{
+		[PaletteDefinition]
```

Comment:
Is this Reference -> Definition change intentional?

Reply:
No, this was a mistake. I must have miss-copied when moving this above the desc.

Good spotting!